db/migrate/20230313104452_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password
+      t.string :phone
```

Comment:
Why is phone number string here? Should `012asb567s` be allowed as phone_number?

Reply:
It shouldn't. I have validate that using `numericality: {only_integer:true}` in `app/model/user.rb` file.

Reply:
apu, used phony_rails gem to validate phone number Thanks.